docs/sources/source/asr/tutorial.rst

```diff
@@ -260,19 +267,19 @@ Enabling multi-GPU training with NeMo is easy:
 
 .. code-block:: bash
 
-    python -m torch.distributed.launch --nproc_per_node=<num_gpus> <nemo_git_repo_root>/examples/asr/jasper.py ...
+    python -m torch.distributed.launch --nproc_per_node=<num_gpus> <nemo_git_repo_root>/examples/asr/quartznet.py ...
 
 
 Large Training Example
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Please refer to the `<nemo_git_repo_root>/examples/asr/jasper.py` for comprehensive example. It builds one train DAG and up to three validation DAGs to evaluate on different datasets.
+Please refer to the `<nemo_git_repo_root>/examples/asr/quartznet.py` for comprehensive example. It builds one train DAG and up to three validation DAGs to evaluate on different datasets.
```

Comment:
"and up to three validation DAGs to evaluate on different datasets." -> "and **uses** up to three validation DAGs to evaluate on different datasets."

But aside of that - what does it mean? How many Validation DAGs do we have in that script? Cannot we have four?

Reply:
Updated

Reply:
just noticed: "builds one trains"